spec/controllers/leaves_controller_spec.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe LeavesController, type: :controller do
+  describe 'GET #index' do
+    it 'returns http success' do
+      get :index
+      expect(response).to be_successful
```

Comment:
Nie za dużo tutaj mamy, powinniśmy sprawdzać czy otrzymaliśmy odpowiednie rekordy.

Dotyczy to również pozostałych testów.